### DISCUSSION TITLE
Web Audio API issue

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9692] [CLI - Capacitor / Cordova modes] [answered]
I am having a **Web Audio API** issue when using the app in the production build. I use the API for **synchronizing** the clock & the sounds.

During the development phase (quasar dev -m cordova -T ios) everything is smooth and synchronized, when I build the app in the prod mode (quasar build -m cordova -T ios), suddenly the sync is gone and everything is all around the place. Any suggestions?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@i-mage 

a bit old but maybe still useful article:
https://www.airpair.com/ionic-framework/posts/using-web-audio-api-for-precision-audio-in-ionic

and some more recent:
https://masteringionic.com/blog/2017-08-02-using-the-web-audio-api-with-ionic/
https://avladov.com/blog/206/using-createjs-webaudio-phonegap-cordova

**Reply by i-mage** [reply]
Once I have installed the Ionic webview plugin (https://quasar.dev/quasar-cli/developing-cordova-apps/preparation#switching-to-ios-wkwebview) , everything is smooth! 

Quite a relief. :-)

Thanks for the suggestions, I will definitely have a look!

**Reply by ontwikkelfabriek** [reply]
@i-mage 

A ok very good.  You have to switch to `wkwebview`  anyways to be able to pass Apple's app review.

**Reply by ontwikkelfabriek** [reply]
@i-mage 

Just for my clarity, could you tell me what version of `cordova-ios` your are using?

**Reply by i-mage** [reply]
6.2.0
